concrete-csprng/src/counter/mod.rs

```diff
@@ -36,6 +36,10 @@ pub struct AesCtr(pub u128);
 #[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
 pub struct ByteCtr(pub u8);
 
+/// Represents the step used to increment the counter used to index on the batch-generated bytes.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
+pub struct StepCtr(pub u8);
```

Comment:
This is not really a counter for me. At least not in the same sense as `ByteCtr`. Maybe a name like `StepSize`  or `Stride` would be more appropriate ?